include/openssl/core_names.h

```diff
@@ -42,8 +42,7 @@ extern "C" {
 
 /* digest parameters */
 #define OSSL_DIGEST_PARAM_XOFLEN    "xoflen"
-#define OSSL_DIGEST_PARAM_CMD       "cmd"
-#define OSSL_DIGEST_PARAM_MSG       "msg"
+#define OSSL_DIGEST_PARAM_SSL3_MS   "ssl3-ms"
```

Comment:
Side note:  I made this name up, and am wondering if it's good enough.  It's currently highly internal, but I'm thinking that if anyone else wants to implement something like this, we might want to have something more descriptive.  Do we care much?  Maybe this is also a cue for implementing the SSL/TLS MAC differently, and this might go away at some point.  I actually don't know, but maybe we should make a collective mental note of this?

Reply:
Its fine, but maybe ssl3ms?  nit.